src/p2p/behaviour.rs

```diff
@@ -18,7 +18,7 @@ use tokio::prelude::*;
 #[derive(NetworkBehaviour)]
 pub struct Behaviour<TSubstream: AsyncRead + AsyncWrite, TSwarmTypes: SwarmTypes> {
     mdns: Mdns<TSubstream>,
-    kademlia: Kademlia<TSubstream>,
+    kademlia: Kademlia<TSubstream, MemoryStore>,
```

Comment:
I tried to make this generic over which RecordStore but that carries a lifetime so I couldn't see a way.